pkgs/development/tools/misc/elfutils/default.nix

```diff
@@ -52,17 +54,25 @@ stdenv.mkDerivation rec {
     })
   ] ++ lib.optional stdenv.hostPlatform.isMusl [ ./musl-error_h.patch ];
 
+  outputs = [ "bin" "dev" "out" "man" ];
+
   hardeningDisable = [ "format" ];
 
   # We need bzip2 in NativeInputs because otherwise we can't unpack the src,
   # as the host-bzip2 will be in the path.
   nativeBuildInputs = [ m4 bison flex gettext bzip2 ]
-    ++ lib.optional stdenv.hostPlatform.isMusl [ pkg-config autoreconfHook ];
+    ++ lib.optional stdenv.hostPlatform.isMusl [ pkg-config autoreconfHook ]
+    ++ lib.optional enableDebuginfod [ pkg-config ];
```

Comment:
This should be lib.optionals and enabling enableDebuginfod and musl will not work because inputs are duplicated.

Reply:
good catch, fixed in  #117599